src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public static void doWork() {
+        System.out.println("Truck started its work");
+
+    }
+
+    public static void stopWork() {
+        System.out.println("Truck stopped working ");;
```

Comment:
The message printed here should also be more informative and indicate that the 'Truck' has stopped working. Additionally, there is an extra semicolon at the end of the line which should be removed.